core/bones/base.py

```diff
@@ -882,3 +951,21 @@ def iter_bone_value(
                     yield idx, None, val
             else:
                 yield None, None, value
+
+    def _compute(self, skel: 'viur.core.skeleton.SkeletonInstance', name: str):
+
+        if "skel" not in inspect.signature(self.compute.fn).parameters:
+            data = self.singleValueUnserialize(self.compute.fn())  # call without any arguments
+        else:
+            skel = skel.clone()
+            skel[name] = None  # we must remove our bone because of recursion
+            data = self.singleValueUnserialize(self.compute.fn(skel=skel))
+
+        if not self.compute.raw:
+            errors = self.fromClient(skel, name, {name: data})
+            if errors is None:
+                return skel[name]
+            else:
+                logging.error(f"Parse Data failed with {errors} \nWe return the raw data.")
```

Comment:
I this line break intended? I'm not sure if the google logging console displays line breaks reasonable.

```suggestion
                logging.error(f"Parse Data failed with {errors}. We return the raw data.")
```